cdap-archetypes/cdap-mapreduce-archetype/README.rst

```diff
@@ -0,0 +1,54 @@
+===========================================================
+Cask Data Application Platform (CDAP) Application Archetype
+===========================================================
+
+This directory contains a CDAP Application Archetype for the Cask Data Application Platform (CDAP)
+that includes a MapReduce program.
+
+
+Installing
+==========
+
+To install the archetype, enter in the CDAP root directory::
+
+  $ mvn clean install -f cdap-archetypes/cdap-mapreduce-archetype/pom.xml
+
+
+Creating
+========
+
+To create a project from the archetype, use this script as an example
+(substituting for ${project.version} and ${your.hadoop.version} as appropriate)::
+
+  mvn archetype:generate 					
+    -DarchetypeGroupId=co.cask.cdap 			
+    -DarchetypeArtifactId=cdap-mapreduce-archetype
+    -DarchetypeVersion=${project.version}
```

Comment:
fixed. Actually this was copied from the cdap-app-archetype and I am fixing it there, too.